integration/types/datatype_test.rb

```diff
@@ -80,15 +80,18 @@ def test_can_insert_all_primitive_datatypes_nil_values
 
       # Create the input
       params = [0]
-      DatatypeUtils.primitive_datatypes.each do |datatype|
+      DatatypeUtils.primitive_datatypes.each do
         params.push(nil)
       end
 
       # Insert into table
       parameters = ["zz"]
-      parameters.push(('a'..'o').to_a)
+      DatatypeUtils.primitive_datatypes.zip('a'..'z') do |datatype, letter|
+        parameters.push(letter)
+      end
+
       arguments = []
-      16.times { arguments.push('?') }
+      (DatatypeUtils.primitive_datatypes.size+1).times { arguments.push('?') }
```

Comment:
For readability, please put spaces before and after the +. I think RubyMine's code formatter is configured by default to add that white-space.